plugin/core/sessions.py

```diff
@@ -1704,12 +1704,17 @@ def notify_plugin_on_session_buffer_change(self, session_buffer: SessionBufferPr
         if self._plugin:
             self._plugin.on_session_buffer_changed_async(session_buffer)
 
-    def _maybe_resolve_code_action(self, code_action: CodeAction) -> Promise[Union[CodeAction, Error]]:
-        if "edit" not in code_action and self.has_capability("codeActionProvider.resolveProvider"):
-            # TODO: Should we accept a SessionBuffer? What if this capability is registered with a documentSelector?
-            # We must first resolve the command and edit properties, because they can potentially be absent.
-            request = Request("codeAction/resolve", code_action)
-            return self.send_request_task(request)
+    def _maybe_resolve_code_action(self, code_action: CodeAction, view: Optional[sublime.View]) -> Promise[Union[CodeAction, Error]]:
```

Comment:
```suggestion
    def _maybe_resolve_code_action(
        self,
        code_action: CodeAction,
        view: Optional[sublime.View]
    ) -> Promise[Union[CodeAction, Error]]:
```

Reply:
Or even this since we tend to use more compact pattern when all arguments fit on a line.

```suggestion
    def _maybe_resolve_code_action(
        self, code_action: CodeAction, view: Optional[sublime.View]
    ) -> Promise[Union[CodeAction, Error]]:
```